docs/man/rpm-config.5.scd

```diff
@@ -0,0 +1,217 @@
+RPM-CONFIG(5)
+
+# NAME
+*rpm-config* - rpm runtime macro configuration
+
+# SYNOPSIS
+_NAME_ _VALUE_
+
+# DESCRIPTION
+The primary configuration mechanism in *rpm* is via macros. On startup,
+*rpm* reads a set of macro files (*rpm-macrofile*(5)) as defined
+in the macro path.
```

Comment:
I feel like "macro path" should be emphasized (underlined) here as it's basically a term that's later referred to in the text.

Reply:
I agree it probably should be emphasized somehow, just wondering about the form. Is there a guidance in man-pages(7) on that or...? (not finding anything immediately obvious here)

Reply:
But that's "expressions", perhaps in the programming sense of the word (?)

Reply:
Hmm, there's a more accurate example perhaps, see the `NULL, NUL, null pointer, and null byte` section in man-pages(7). It defines the term "null pointer" and its italicized there.

Reply:
Added underlining now, both for macro path and rpmrc path (which is very similar)